js/src/workers/ComputeAndInjectDiffsWorker.js

```diff
@@ -89,13 +89,17 @@ function computeAndInjectDiffs({ previousData, currentData }) {
   const currentImageData = imageTo2DArray(
     currentData, maxWidth - currentData.width);
 
+  self.postMessage({ progress: 30 });
```

Comment:
Yeah, not sure if this is worth the effort. I think I'm just going to show a "spinner" type indicator (not necessarily a spinning thing, but you get the point).